catala/cantar/cantar.md

```diff
@@ -1,3 +1,6 @@
 La pluja que cau sobre la tenda et desperta. 
-El teu company de tenda havia deixat una bossa plena de llaminadures 
-prop del teu cap, deu ser això el que t'ha fet somniar amb les parets de sucre...
\ No newline at end of file
+El teu company de tenda havia deixat una bossa plena de llaminadures prop del teu cap, deu ser això el que t'ha fet somniar amb les parets de sucre...
+
+I ara... què vols fer?
+
+[Tasto una llaminadura] (cantar/tastar/tastar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.